tests/testthat/test-num-poisson_log_loss.R

```diff
@@ -0,0 +1,20 @@
+count_results <- data_counts()$basic
+count_missing <- data_counts()$missing
+count_poor    <- data_counts()$poor
```

Comment:
Can you please put these inside the `test_that()` block so they don't persist between files?